docs/app/Examples/modules/Checkbox/Variations/Fitted.js

```diff
@@ -6,13 +6,13 @@ export default class CheckboxFittedExample extends Component {
     return (
       <div>
         <Segment className='compact'>
-          <Checkbox className='fitted' />
+          <Checkbox />
         </Segment>
         <Segment className='compact'>
-          <Checkbox className='fitted slider' />
+          <Checkbox className='slider' />
         </Segment>
         <Segment className='compact'>
-          <Checkbox className='fitted toggle' />
+          <Checkbox className='toggle' />
```

Comment:
The fitted class is automatically added.  Note the addition of the message explaining so on the example:

![image](https://cloud.githubusercontent.com/assets/5067638/10743541/07d9107e-7bf1-11e5-9a13-23b24e972281.png)